src/downshift.js

```diff
@@ -633,7 +633,7 @@ class Downshift extends Component {
     if (!uiDescriptor) {
       return null
     }
-    // doing React.Children.only for Preact support ⚛️
+    // doing React.Children.only for Preact support 
```

Comment:
Sorry as it was a really easy change I edited via Github directly, seems like the emoji goes away for some reason?